tests/e2e/volume_health_test.go

```diff
@@ -82,23 +82,27 @@ var _ = ginkgo.Describe("Volume health check", func() {
 			deleteResourceQuota(client, namespace)
 		}
 		if pvc != nil {
-			ginkgo.By("checking host status")
-			err := waitForHostToBeUp(hostIP)
-			time.Sleep(pollTimeoutShort)
-			if err != nil {
+			if hostIP != "" {
+				ginkgo.By("checking host status")
+				err := waitForHostToBeUp(hostIP)
 				time.Sleep(pollTimeoutShort)
+				if err != nil {
+					time.Sleep(pollTimeoutShort)
+				}
 			}
-			err = fpv.DeletePersistentVolumeClaim(client, pvc.Name, namespace)
+			err := fpv.DeletePersistentVolumeClaim(client, pvc.Name, namespace)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}
 		if pvclaim != nil {
-			ginkgo.By("checking host status")
-			err := waitForHostToBeUp(hostIP)
-			time.Sleep(pollTimeoutShort)
-			if err != nil {
+			if hostIP != "" {
+				ginkgo.By("checking host status")
+				err := waitForHostToBeUp(hostIP)
 				time.Sleep(pollTimeoutShort)
+				if err != nil {
+					time.Sleep(pollTimeoutShort)
+				}
```

Comment:
This static sleep is not required now, since we are calling the method which polls for the host status, hence removed this block of code from other places also.